exporters/cinder.go

```diff
@@ -21,43 +20,11 @@ type CinderExporter struct {
 	BaseOpenStackExporter
 }
 
-var volume_status = []string{
-	"creating",
-	"available",
-	"reserved",
-	"attaching",
-	"detaching",
-	"in-use",
-	"maintenance",
-	"deleting",
-	"awaiting-transfer",
-	"error",
-	"error_deleting",
-	"backing-up",
-	"restoring-backup",
-	"error_backing-up",
-	"error_restoring",
-	"error_extending",
-	"downloading",
-	"uploading",
-	"retyping",
-	"extending",
-}
-
-func mapVolumeStatus(volStatus string) int {
-	for idx, status := range volume_status {
-		if status == strings.ToLower(volStatus) {
-			return idx
-		}
-	}
-	return -1
-}
-
 var defaultCinderMetrics = []Metric{
 	{Name: "volumes", Fn: ListVolumes},
 	{Name: "snapshots", Fn: ListSnapshots},
 	{Name: "agent_state", Labels: []string{"uuid", "hostname", "service", "adminState", "zone", "disabledReason"}, Fn: ListCinderAgentState},
-	{Name: "volume_status", Labels: []string{"id", "name", "status", "bootable", "tenant_id", "size", "volume_type", "server_id"}, Fn: nil},
+	{Name: "volume_gb", Labels: []string{"id", "name", "status", "bootable", "tenant_id", "size", "volume_type", "server_id"}, Fn: nil},
```

Comment:
I think the label "size" ist not neede wens size is already in the metric.

Reply:
you are right. 
Below commit remove it.